website/docs/docs/build/unit-tests.md

```diff
@@ -29,6 +29,7 @@ With dbt Core v1.8 and dbt Cloud environments that have gone versionless by sele
 - Unit tests must be defined in a YML file in your `models/` directory.
 - Table names must be [aliased](/docs/build/custom-aliases) in order to unit test `join` logic.
 - Redshift customers need to be aware of a [limitation when building unit tests](/reference/resource-configs/redshift-configs#unit-test-limitations) that requires a workaround. 
+- All references (`ref()`) used in your model must be included in the unit test configuration as input fixtures, even if they do not directly affect the logic being tested. If these references are missing, you may encounter "node not found" errors during compilation.
```

Comment:
Should be [`ref`](https://docs.getdbt.com/reference/dbt-jinja-functions/ref) _or_ [`source`](https://docs.getdbt.com/reference/dbt-jinja-functions/source).

See https://github.com/dbt-labs/docs.getdbt.com/issues/6331 for a related docs issue asking for a unified super concept that describes both `ref` and `source` calls.